os_build_stories/raspbian_9_stretch

```diff
@@ -0,0 +1,54 @@
+cmake:
+
+    sudo apt-get install cmake
+
+xsltproc:
+
+    sudo apt-get install xsltproc
+
+tidy-html5:
+
+    git clone git://github.com/htacg/tidy-html5
+    cd tidy-html5/build/cmake
+    cmake ../..
+    make
+    sudo make install
```

Comment:
debian has libtidy5 and libtidy-dev packages you can use, you don't need to build from source, it should be up to date enough